wdio.conf.js

```diff
@@ -1,14 +1,52 @@
 require('dotenv').config();
 require('babel-register');
 
+const os = require('os');
+
+// eslint-disable-next-line prefer-destructuring
+const argv = require('yargs').argv;
+
+const uuid = require('uuid');
+
+const {inject} = require('./scripts/tests/openh264');
+
+const browser = process.env.BROWSER || 'chrome';
+const tunnelId = uuid.v4();
+const port = process.env.PORT || 4567;
+const {suite} = argv;
```

Comment:
If suite isn't defined, we should give it a default value via yargs. Maybe just "all"

Reply:
good point, will add that!